modules/custom-post-types/portfolios.php

```diff
@@ -563,11 +564,13 @@ static function portfolio_shortcode_html( $atts ) {
 
 				<?php
 				// The content
-				if ( false != $atts['display_content'] ): ?>
-					<div class="portfolio-entry-content"><?php the_excerpt(); ?></div>
-				<?php endif; ?>
-				</div><!-- close .portfolio-entry -->
-			<?php
+				if ( false !== $atts['display_content'] ): 
++							if ( true == $atts['show_excerpt'] ) { ?>
++								<div class="portfolio-entry-content"><?php the_excerpt(); ?></div><!-- close .portfolio-entry -->
++							<?php } else { ?>
++										<div class="portfolio-entry-content"><?php the_content(); ?></div><!-- close .portfolio-entry -->
++								<?php }
```

Comment:
It seems weird that we're using both `:`/`endif;`, and braces within the same conditional.  Let's stick to one syntax.